pkg/apiserver/registry/stats/nodelatencystats/rest.go

```diff
@@ -50,38 +57,80 @@ func (r *REST) Destroy() {
 }
 
 func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStats{}, nil
+	summary := obj.(*statsv1alpha1.NodeLatencyStats)
+	err := r.indexer.Update(summary)
+	if err != nil {
+		return nil, errors.NewInternalError(err)
+	}
+
+	return summary, nil
 }
 
 func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStats{}, nil
+	obj, exists, err := r.indexer.GetByKey(name)
+	if err != nil || !exists {
+		return nil, errors.NewNotFound(statsv1alpha1.Resource("nodelatencystats"), name)
+	}
+
+	return obj.(*statsv1alpha1.NodeLatencyStats), nil
 }
 
 func (r *REST) NewList() runtime.Object {
 	return &statsv1alpha1.NodeLatencyStats{}
 }
 
 func (r *REST) List(ctx context.Context, options *internalversion.ListOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStatsList{}, nil
+	objs := r.indexer.List()
+	if objs == nil {
+		return &statsv1alpha1.NodeLatencyStatsList{}, nil
+	}
+
+	entries := make([]statsv1alpha1.NodeLatencyStats, 0, len(objs))
+	for _, obj := range objs {
+		entries = append(entries, *obj.(*statsv1alpha1.NodeLatencyStats))
+	}
+
+	return &statsv1alpha1.NodeLatencyStatsList{Items: entries}, nil
 }
 
 func (r *REST) ConvertToTable(ctx context.Context, obj runtime.Object, tableOptions runtime.Object) (*metav1.Table, error) {
-	// TODO: fill this function in next PR
 	table := &metav1.Table{
 		ColumnDefinitions: []metav1.TableColumnDefinition{
 			{Name: "SourceNodeName", Type: "string", Format: "name", Description: "Source node name."},
-			{Name: "NodeLatencyStatsList", Type: "array", Format: "string", Description: "Node IP latency list."},
+			{Name: "PeerNodeLatencyStats", Type: "array", Format: "string", Description: "Current Node to each peers latency."},
```

Comment:
This can be a really long list (think of a cluster with 100+ Nodes), so I don't know if this is the best way to handle this. That being said, I guess there is a similar situation with `Endpoints` when running `kubectl get svc`. Let's see what an example looks like (see my comment above).
cc @tnqn

Reply:
Yes, that make sense, maybe I need to add `Limit` and `Continue` defined in `k8s.io/apimachinery@v0.29.2/pkg/apis/meta/internalversion/types.go` and return paged data.

Maybe I need to look at getting the paged data form metatable.

Reply:
Does it really support "array", I don't find this type in https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#data-types.

Reply:
@tnqn Yes, in `/go/pkg/mod/k8s.io/apimachinery@v0.29.2/pkg/apis/meta/v1/types.go#1349` annotation, it shows that `arrary` is support, I don't know why it is not support in your link.

![image](https://github.com/antrea-io/antrea/assets/47499836/3cb95db5-2d10-4dd3-a8e2-6a5d868de222)

Reply:
A deeper into the link, it appears to provide annotations for arrays in the swagger openapi, https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#dataTypeType